src/components/Scheme/FixedReputationAllocationScheme.tsx

```diff
@@ -0,0 +1,159 @@
+import { Address, ISchemeState, } from "@daostack/client";
+import { checkWeb3ProviderAndWarn, getArc } from "arc";
+import { ErrorMessage, Field, Form, Formik, FormikProps } from "formik";
+import { schemeName} from "lib/util";
+import * as React from "react";
+import { BreadcrumbsItem } from "react-breadcrumbs-dynamic";
+import * as Sticky from "react-stickynode";
+import { connect } from "react-redux";
+import { IRootState } from "reducers";
+import { showNotification } from "reducers/notifications";
+import * as schemeCss from "./Scheme.scss";
+import * as css from "./FixedReputationAllocationScheme.scss";
+
+interface IStateProps {
+  currentAccountAddress: Address;
+  daoAvatarAddress: Address;
+  scheme: ISchemeState;
+}
+
+interface IDispatchProps {
+  showNotification: typeof showNotification;
+}
+
+const mapDispatchToProps = {
+  showNotification,
+};
+
+type IProps = IStateProps & IDispatchProps;
+
+const mapStateToProps = (state: IRootState, ownProps: any) => {
+  return {...ownProps,
+    currentAccountAddress: state.web3.currentAccountAddress
+  };
+};
+
+interface IFormValues {
+  thisAccount: boolean;
+  accountAddress: string;
+}
+
+class FixedReputationAllocationScheme extends React.Component<IProps, null> {
+
+  public async handleSubmit(values: IFormValues, { _props, setSubmitting, _setErrors }: any): Promise<void> {
+    if (!(await checkWeb3ProviderAndWarn(this.props.showNotification.bind(this)))) { return; }
+
+    // TODO: some call in the subgraph?
+  }
+
+  public render() {
+    const { currentAccountAddress, daoAvatarAddress, scheme } = this.props;
+    const arc = getArc();
+
+    return (
+      <div className={schemeCss.schemeContainer}>
+        <BreadcrumbsItem to={`/dao/${daoAvatarAddress}/scheme/${scheme.id}`}>{schemeName(scheme, scheme.address)}</BreadcrumbsItem>
+
+        <Sticky enabled top={50} innerZ={10000}>
+          <h2 className={schemeCss.schemeName}>
+            {schemeName(scheme, scheme.address)}
+          </h2>
+        </Sticky>
+
+        <div>
+          <h3>100 Rep to redeem </h3>
+          <b>Which account would you like to receive the reputation?</b>
```

Comment:
I wonder if we should have a more detailed helpful description for the user of what is going on here, why they are being presented with this UI?

Reply:
that would be up to @A-Zak